utils/markdown/blocks.go

```diff
@@ -130,13 +130,13 @@ func blockStart(markdown string, indentation int, r Range, matchedBlocks, unmatc
 		return nil
 	}
 
-	if start := blockQuoteStart(markdown, indentation, r, matchedBlocks, unmatchedBlocks); start != nil {
+	if start := blockQuoteStart(markdown, indentation, r); start != nil {
```

Comment:
@hmhealey - Are they good to be removed? Or do we actually need them?

Reply:
I think they might have been there since the code paralleled `listStart` and `indentedCodeStart`, but I'm fine removing them for simplicity.